Woodall_Robert/Assignments/registration_form/templates/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Registration Form</title>
+	<style type="text/css">
+		.error{
+			color: red;
+			font-weight: bold;
+		}
+	</style>
+</head>
+<body>
+    <form action="/process" method='POST'>
+        <div>
+	        <label>Email:</label>
+	        <input type="text" name="email">
+	    </div>
+	    <div>
+	        <label>First Name:</label>
+	        <input type="text" name="fname">
+	    </div>
+	    <div>
+	        <label>Last Name:</label>
+	        <input type="text" name="lname">
+	    </div>
+	    <div>
+	        <label>Password:</label>
+	        <input type="text" name="pw">
+	    </div>
+	    <div>
+	        <label>Confirm Password:</label>
+	        <input type="text" name="confirm_pw">
+	    </div>
+	    <input type="submit" name="Submit">
+    </form>
+    {% with messages = get_flashed_messages() %}
```

Comment:
it's a better workflow to display form errors on the top of the page rather than on the bottom of the page. other when the user is redirected back to this page, they won't see the errors until they scroll down, which might cause some initial confusion for them